tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "moduleResolution": "node",
     "outDir": "./dist",
     "sourceMap": true,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
this is a deprecated option so tsc was complaining but we weren't using it anyway